collector/sbd/sbd.go

```diff
@@ -35,8 +35,7 @@ func NewCollector(sbdPath string, sbdConfigPath string) (*sbdCollector, error) {
 	}
 
 	c.SetDescriptor("devices", "SBD devices; one line per device", []string{"device", "status"})
-	c.SetDescriptor("watchdog_timeout", "sbd watchdog timeout", []string{"device"})
-	c.SetDescriptor("msgwait_timeout", "sbd msgwait timeout", []string{"device"})
+	c.SetDescriptor("sbd_timeouts", "sbd watchdog timeout", []string{"device", "type"})
```

Comment:
The description is a bit misleading, as we have both watchdog and msgwait timeouts, not just the watchdog one. Also we dont need the `sbd_` prefix, that's already added by the constructor.
Here is something more generic:
```suggestion
	c.SetDescriptor("timeouts", "SBD timeouts for each device and type", []string{"device", "type"})
```